lib/digsync_web/live/group_details_live.ex

```diff
@@ -1,14 +1,27 @@
 defmodule DigsyncWeb.GroupDetailsLive do
   use DigsyncWeb, :live_view
   alias Digsync.Accounts.Group
+  alias Digsync.Accounts.Groups
 
-  def mount(_params, _session, socket) do
-    {:ok, socket}
+  def mount(%{"group_id" => group_id}, _session, socket) do
+    IO.puts("group id is #{group_id}")
```

Comment:
Same thing can be achieved by doing IO.inspect(group_id, label: "group id is") 
There's also some new elixir feature I havn't even googled or looked at called "dbg" may be worth taking a look